man/wacom.man

```diff
@@ -235,7 +235,10 @@ sets the list of serial numbered devices that need to be hotplugged for a physic
 device. The 'type' option may be any of "pen", "airbrush", "artpen", or "cursor".
 This option is only available on wacom V devices (Intuos series and Cintiq 21U).
```

Comment:
heh, I'd say that sentence is a bit ouf of date now...

Reply:
Such is the nature of documentation ;-)

Reply:
Would you like me to do something about it? I don't know what additional hardware it applies to...

Reply:
Yes please. "Intuos Pro and Cintiq series" I think is good enough here, @jigpu?

Reply:
Tweaked, I did not include "Pro" as that's a distinct product line from the Intuos3 I have that supports it.

Reply:
The intuos3 at this point is 12 years old? And quite a few years ago, Wacom changed the naming, so what used to be Bamboos are now called Intuos and the previous Intuos devices are called Intuos Pro. And the Bamboos don't support serials, so basically: if you leave out the "Pro", it's the wrong device for anything released in the last 8 or so years. And no, don't ask me **why** the naming was changed in the most confusing way possible.

Reply:
If the feature is only dependent on the device being able to identify tool serials... should we remove that line entirely and replace it with something like "This feature is only supported if xsetwacom can report a tool serial number from the device"?

Reply:
yeah, that works too. I'm not a big fan of the emphasis on `xsetwacom` though since you can get the same information from e.g. `xinput`. But I guess it's less confusing than writing something more generic...

Reply:
I agree with simplifying the wording, but would want to keep the Cintiq and Intuos Pro examples since most people probably have no clue which devices support tool IDs. Also, the phrase "wacom V devices" is meaningless to anyone who doesn't maintain this driver, and probably even incorrect (I'd bet the feature also works with AES pens that report serial numbers) so I'd remove it as well.

> And no, don't ask me why the naming was changed in the most confusing way possible.

It's a mystery to us all :dizzy_face: